meerk40t/core/bindalias.py

```diff
@@ -124,7 +124,7 @@
     ),
     "ctrl+a": ("element* select",),
     "ctrl+c": ("clipboard copy",),
-    "ctrl+e": ("circle 500 500 500",),
+    "ctrl+e": ("circle 0.5in 0.5in 0.5in stroke red classify",),
```

Comment:
Please see the comments earlier in the file - you need to insert a new entry before the previous one so this should read:
```
    "ctrl+e": ("circle 0.5in 0.5in 0.5in stroke red classify","circle 500 500 500",),
```
The old versions are used to provide automated update if the user has not changed the definition themselves.

Reply:
Done